.github/dependabot.yml

```diff
@@ -31,6 +31,10 @@ updates:
         - "glib*"
         - "gobject*"
         - "gstreamer*"
+    serde-related:
```

Comment:
This list is in alphabetical order. Please move the `serde-related` group under the `objc2-related` group.